src/composables/useUtilities.ts

```diff
@@ -242,12 +242,33 @@ export default function useUtilities() {
     return JSON.parse(JSON.stringify(val))
   }
 
+  /**
+   * For the label of a `required` input.
+   * If the field is required and the label ends with an '*', strip it out.
+   *
+   * @param label the label to strip
+   * @param required whether or not the field is required
+   * @returns the stripped label
+   */
+  const stripRequiredLabel = (label: string, required: boolean): string => {
+    if (!required) {
+      return label
+    }
+
+    if (label.match(/( )?\*$/gi)) {
```

Comment:
You need to force a boolean here so that you don't pass with an empty array
```suggestion
    if (!!label.match(/( )?\*$/gi)) {
```

Reply:
I tried it like this but I get a TS warning:

![image](https://user-images.githubusercontent.com/67973710/231165337-ff289a3b-db99-45e5-9a64-e8302f6abfe5.png)

Reply:
Ah do this then instead
```
Boolean(label.match(/( )?\*$/gi))
```

Reply:
heh, nope, can't get around it that way either:

![image](https://user-images.githubusercontent.com/67973710/231169244-b6ebdefc-dc1f-4f2e-85dd-dde5158b3206.png)

Reply:
lol ok then let's do this:

```ts
/( )?\*$/gi.test(label)
```

Reply:
![image](https://user-images.githubusercontent.com/67973710/231179527-b011dee4-9e32-4521-a966-04405342b5df.png)